twitter/tests/test.bal

```diff
@@ -193,8 +193,10 @@ function testGetFollowers() {
     var userResponse = twitterClient->getFollowers(userID);
 
     if (userResponse is error) {
+        log:printInfo(userResponse.toString());
```

Comment:
We don't need to log info inside test cases.

Reply:
Removed those